### PR TITLE
Enable 4.11 releases

### PR DESCRIPTION
**Pull request by mukrishn**
Manifest changes to include new nightly releases for managed service clusters.

### CODE REVIEW

**Comment by mukrishn** [comment]
Also downgraded gcloud sdk to 367.0.0 to workaround this issue https://issuetracker.google.com/issues/209655844 and tested!

**Review by smalleni — APPROVED** [review]

File: dags/openshift_nightlies/manifest.yaml [hunk 1]
```diff
@@ -15,6 +15,10 @@ versions:
   alias: future
   releaseStream: 4.10.0-0.nightly
   baremetalReleaseStream: latest-4.10
+- version: 4.11
+  alias: future
```
Comment (dry923) [inline dags/openshift_nightlies/manifest.yaml:19]: Doesn't this leave us w/ 2 versions flagged as future? Will that break things?

Comment (mukrishn) [reply dags/openshift_nightlies/manifest.yaml]: already done!

**Review by dry923 — APPROVED** [review]
LGTM!

File: dags/openshift_nightlies/manifest.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ dagConfig:
 
 platforms:
   cloud:
-    versions: [4.9, "4.10"]
+    versions: [4.9, "4.10", 4.11]
```
[the remark this answers is not in the document]
Comment (mukrishn) [reply dags/openshift_nightlies/manifest.yaml:31]: @morenod thanks for looking at it, I think its required only for `4.10`, `4.20` and so on as python is not counting the last digit `0` and read it as `4.1` and `4.2`. 
I tested this and was able to generate dags for `4.11`.

[the remark this answers is not in the document]
Comment (mukrishn) [reply dags/openshift_nightlies/manifest.yaml:31]: and in the manifest file, it is referred without quotes already [here](https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/manifest.yaml#L10-L13) as well.

[the remark this answers is not in the document]
Comment (morenod) [reply dags/openshift_nightlies/manifest.yaml:31]: ok, then it LGTM!

**Review by morenod — APPROVED** [review]
LGTM

**Pull request merged**